README.md

```diff
@@ -113,7 +113,7 @@ Other commonly-used configuration options are:
 
 Many of these have been mentioned previously. 
 
-**NOTE:** It is important to ensure the `mcuboot` configuration parameters are **the same** for the bootloader and the bootable application! Also `mcuboot.header-size` has been deprecated as of mcuboot v1.7 and onward.
+**NOTE:** It is important to ensure the `mcuboot` configuration parameters are **the same** for the bootloader and the bootable application! Also `mcuboot.header-size` has been deprecated as of mcuboot v1.7 and onward. We no-longer need to set it in mbed_app.json file. But we still need to note down the planned header-size and pass it to imgtool when signing built images.
```

Comment:
Actually.. I am working on an update to CMake so that it executes imgtool automatically, using the header size set in mbed_app.json.  So I think we may actually want to keep this setting here.

Reply:
Could you try to calculate  header size from `mcuboot.primary-slot-address' and `"target.mbed_app_start`? If we add `mcuboot.header-size` in mbed_app.json/json5 file, there will be error when compiling at least this is what happens in mbed studio.

Reply:
I think that the ship has already sailed on keeping compatibility with Mbed Studio, as Mbed CE uses a totally different memory bank system and doesn't use mbed_app_start.  What's the error with enabling header-size?  I don't see it building on my end.

Reply:
Makes sense